src/js/containers/covid19/assistanceListing/SpendingByCFDAContainer.jsx

```diff
@@ -307,7 +315,7 @@ const SpendingByCFDAContainer = ({ activeTab, scrollIntoView }) => {
             fetchSpendingByCfdaCallback();
         }
         changeCurrentPage(1);
-    }, [pageSize, defCodes, sort, order, activeTab, assistanceTotals]);
+    }, [pageSize, defCodes, sort, order, activeTab, query, assistanceTotals]);
```

Comment:
i have a pr that will conflict with this line; just removing assistanceTotals FYSA